src/control/server/harness.go

```diff
@@ -186,22 +186,10 @@ func (h *EngineHarness) CallDrpc(ctx context.Context, method drpc.Method, body p
 	// the first one that is available to service the request.
 	// If the request fails, that error will be returned.
 	for _, i := range h.Instances() {
-		if !i.IsReady() {
-			if i.IsStarted() {
-				if err == nil {
-					err = errInstanceNotReady
-				}
-			} else {
-				if err == nil {
-					err = FaultDataPlaneNotStarted
-				}
-			}
```

Comment:
I removed it because I don't think there is significant benefit in performing the check, CallDrpc now will only return EngineNotReady or DRPCNotReady, returning FaultDataPlaneNotStarted is less valuable because the description is "not started or not responding" which a less specific error. Is "not started" vs just "not ready" something that you feel is particularly useful in this context?

Reply:
I don't feel that strongly about it, but it *could* tell you whether the engines have crashed, or if they just haven't finished coming up yet.